demos/text_to_speech_demo/python/models/mel2wave_ie.py

```diff
@@ -261,7 +272,8 @@ def forward(self, mel):
         c_begin = 0
         c_end = cur_w
         while c_begin < cols:
-            audio = self.exec_net[active_net].infer(inputs={"mel": mel[:, :, c_begin:c_end]})["audio"]
+            self.requests[active_net].infer(inputs={"mel": mel[:, :, c_begin:c_end]})
+            audio = self.requests[active_net].get_tensor("audio").data
```

Comment:
```suggestion
            audio = self.requests[active_net].get_tensor("audio").data[:]
```
The `data` field will be overwritten by the next infer request call and the output collected in `res_audio` will be incorrect, so it's essential to copy tensor data in such cases

Reply:
Done